# main loop frequency [Hz] (should be as fast as F/T sensor publish frequency)
loop_rate: 650.0

# waiting time after moving to each pose before taking F/T measurements
wait_time: 4.0

# Name of the moveit group
moveit_group_name: 'left_arm'

# Name of the calib file
calib_file_name: 'ft_calib_data.yaml'

# Name of the directory
calib_file_dir: '~/.ros/ft_calib'

# Name of file to store measurements
meas_file_name: 'ft_calib_meas.txt'

# Name of the directory
meas_file_dir: '~/.ros/ft_calib'

# don't execute random poses
random_poses: false

# number of random poses
number_random_poses: 0

# use joint positions instead of cartesian poses
use_joints: false

# the poses to which to move the arm in order to calibrate the F/T sensor
# format: [x y z r p y]  in meters, radians
# poses_frame_id sets the frame at which the poses are expressed
poses_frame_id: 'arm_base_link'

pose0: [-0.486, 0.367, -0.000, 1.571, 1.571, 3.130]
pose1: [-0.486, 0.367, -0.000, 1.571, 0.8, 3.140]
pose2: [-0.486, 0.367, -0.000, 1.571, 0.4, 3.140]
pose3: [-0.486, 0.367, -0.000, 1.571, 0.0, 3.140]
pose4: [-0.486, 0.367, -0.000, 1.571, -0.4, 3.140]
pose5: [-0.486, 0.367, -0.000, 1.571, -0.8, 3.140]
pose6: [-0.486, 0.367, -0.000, 1.571, -1.571, 3.140]
pose7: [-0.486, 0.367, -0.000, 2.8, 0.0, 3.140]
pose8: [-0.486, 0.367, -0.000, 2.3, 0.0, 3.140]
pose9: [-0.486, 0.367, -0.000, 2.0, 0.0, 3.140]
pose10: [-0.486, 0.367, -0.000, 1.1, 0.0, 3.140]
pose11: [-0.486, 0.367, -0.000, 0.8, 0.0, 3.140]
pose12: [-0.486, 0.367, -0.000, 0.4, 0.0, 3.140]
pose13: [-0.486, 0.367, -0.000, 0.01, 0.0, 3.140]
pose14: [-0.486, 0.367, -0.000, 3.093, -0.012, -2.467]
pose15: [-0.486, 0.367, -0.000, 2.843, -0.012, -2.467]
pose16: [-0.486, 0.367, -0.000, 2.372, -0.012, -2.467]
pose17: [-0.486, 0.367, -0.000, 2.0, -0.012, -2.467]
pose18: [-0.486, 0.367, -0.000, 1.6, -0.012, -2.467]
pose19: [-0.486, 0.367, -0.000, 1.2, -0.012, -2.467]
pose20: [-0.486, 0.367, -0.000, 0.8, -0.012, -2.467]
pose21: [-0.486, 0.367, -0.000, 0.4, -0.012, -2.467]
pose22: [-0.486, 0.367, -0.000, 0.0, -0.012, -2.467]
pose23: [-0.486, 0.367, -0.000, 2.399, -0.568, 1.058]
pose24: [-0.486, 0.367, -0.000, 2.083, -0.858, 1.531]
pose25: [-0.486, 0.367, -0.000, 1.480, -0.962, 2.280]
pose26: [-0.486, 0.367, -0.000, 0.890, -0.748, 3.043]
pose27: [-0.486, 0.367, -0.000, 0.677, -0.429, -2.865]
pose28: [-0.486, 0.367, -0.000, 0.607, -0.049, -2.576]
pose29: [-0.486, 0.367, -0.000, 0.621, 0.207, -2.396]
pose30: [-0.486, 0.367, -0.000, 2.252, 0.384, -2.121]
pose31: [-0.486, 0.367, -0.000, 2.258, -0.402, -2.726]
pose32: [-0.486, 0.367, -0.000, 2.425, -0.685, -3.043]
pose33: [-0.486, 0.367, -0.000, 2.785, -0.898, 2.742]
pose34: [-0.486, 0.367, -0.000, -2.869, -0.919, 1.958]